styles.css

```diff
@@ -2,21 +2,67 @@
     box-sizing: border-box;
 }
 
-body, html {
+body{
     margin: 0;
     padding: 0;
+    background-color: #424b52;
+    color: whitesmoke;
+    font-family: 'Lato', sans-serif;
+    font-size: 20px;
+    font-weight: 400;
 }
 
-nav {
+/* nav {
     height: 5em;
     width: 100%;
     box-shadow: 0px 5px 8px 5px rgba(0, 0, 0, 0.1);
     display: flex;
     align-items: center;
+} */
+/* ############ THE LIST ########### */
+ul{
```

Comment:
A minor style thing but since I already called out the comments, I tend to be a fan of spaces after the CSS names (`ul {`).

Reply:
Do we have plans to enforce a style guide?

Reply:
> Do we have plans to enforce a style guide?

Presumably at some point in the future, I think they're good ideas. I could see an argument against including them in beginner-focused projects, but I'm not sure I buy those arguments without more consideration.